### DISCUSSION TITLE
Minecraft Editor on other platforms than Windows

### DISCUSSION THREAD

**Discussion by TheMightyDarkness** [Mojang/minecraft-editor#198] [Feedback]
Currently, the Minecraft Bedrock Editor is Windows only. But what if it was available on other platforms such as iOS, Playstation, Xbox, Android, Chromebook?

**Comment by bryhaw** [comment]
Thanks for the suggestion TheMightyDarkness! We have discussed this a bit internally. I think we eventually would love to see Editor cross-platform but we first want to ensure creators find it valuable on Windows. I would love to hear what others think at if they believe there is a need on other platforms.

**Reply by Djwhitelion** [reply]
As someone who builds alot of stuff for the server that my kids play on. Builds mini games etz. I use all my time on the phone or tablet. I only use my computer when i need to code stuff for the plugin i work on for the server. I would love that editor works on android phone as well. I heavily use world edit tool. but at times. it is limited as well. 

so a way to create big structures and add them to other worlds would be great. This way i can create and just work while the kids do there stuff on the server. And i can do some maintenance work at night and add stuff.

**Reply by Landy005** [reply]
> Thanks for the suggestion TheMightyDarkness! We have discussed this a bit internally. I think we eventually would love to see Editor cross-platform but we first want to ensure creators find it valuable on Windows. I would love to hear what others think at if they believe there is a need on other platforms.

I think the problem is just that the audience who would use this on other platforms isn't reached yet.

See > https://github.com/Mojang/minecraft-editor/discussions/200
